src/ua/lviv/iot/algo/part1/lab1/Bicycle.java

```diff
@@ -0,0 +1,81 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.Setter;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+
+public class Bicycle {
+    private static final double priceBike = 10000; //price in UAH
```

Comment:
```suggestion
    private static final double priceIncrementUah = 10000;
```
Make your declaration names self-descriptive, so that you won't have to write comments most of the time